charts/premium/metallb-config/Chart.yaml

```diff
@@ -9,7 +9,7 @@ apiVersion: v2
 appVersion: latest
 dependencies:
   - name: common
-    version: 25.2.4
+    version: 25.2.6
```

Comment:
You can't do that until common is released

Reply:
Thanks for the hint, so this is supposed to be done in a separate pr once it is relesase?

Reply:
You can just wait until renovate gets the update and pushes it to all charts